SConstruct

```diff
@@ -253,7 +253,12 @@ if env_base['platform'] != "":
     selected_platform = env_base['platform']
 elif env_base['p'] != "":
     selected_platform = env_base['p']
-    env_base["platform"] = selected_platform
+
+# Alias "linux" and "bsd" to "linuxbsd"
+if selected_platform == 'linux' or selected_platform == 'bsd':
+    selected_platform = 'linuxbsd'
```

Comment:
Should also alias "x11" to "linuxbsd" so that exiting build scripts continue to work.